src/pages/regulatory/_eu-grid.tsx

```diff
@@ -1,164 +1,240 @@
-    height: 2rem;
-    text-align: center;
-`
-const License = styled.div`
-    margin: 0;
-    display: flex;
-    align-items: center;
-
-    & a {
-        margin-left: 0.3rem;
-        font-size: 1rem;
-        line-height: 1rem;
-        color: var(--color-grey-3);
-        text-decoration: none;
-
-        :hover {
-            color: var(--color-red);
-            text-decoration: underline;
-        }
-    }
-`
-
-const AttachmentIcon = styled(ViewLicense)`
-    margin-left: 1rem;
-
-    path {
-        transition: 0.2s fill;
-    }
-    &:hover {
-        path {
-            fill: red;
-        }
-    }
-`
-
-const GridItem = ({ name, order, url }: GridItemProps) => (
-    <Country>
-        <Order>{order}</Order>
-        <License>
-            <Localize translate_text={name} />
-            <a href={url} target="_blank" rel="noopener noreferrer">
-                <AttachmentIcon />
-            </a>
-        </License>
-    </Country>
-)
 
 const EUgrid = () => {
     return (
-        <CssGrid columns="repeat(2, 1fr)">
-            <CssGridColumn>
-                <GridItem name="_t_Austria_t_" url="/regulatory/countries/Austria.pdf" order="1" />
-                <GridItem
-                    name="_t_Bulgaria_t_"
-                    url="/regulatory/countries/Bulgaria.pdf"
-                    order="2"
-                />
-                <GridItem name="_t_Croatia_t_" url="/regulatory/countries/Croatia.pdf" order="3" />
-                <GridItem name="_t_Cyprus_t_" url="/regulatory/countries/Cyprus.pdf" order="4" />
-                <GridItem
-                    name="_t_Czech Republic_t_"
-                    url="/regulatory/countries/CzechRepublic.pdf"
-                    order="5"
-                />
-                <GridItem name="_t_Denmark_t_" url="/regulatory/countries/Denmark.pdf" order="6" />
-                <GridItem name="_t_Estonia_t_" url="/regulatory/countries/Estonia.pdf" order="7" />
-                <GridItem name="_t_Finland_t_" url="/regulatory/countries/Finland.pdf" order="8" />
-                <GridItem name="_t_France_t_" url="/regulatory/countries/France.pdf" order="9" />
-                <GridItem name="_t_Germany_t_" url="/regulatory/countries/Germany.pdf" order="10" />
-                <GridItem name="_t_Greece_t_" url="/regulatory/countries/Greece.pdf" order="11" />
-                <GridItem name="_t_Hungary_t_" url="/regulatory/countries/Hungary.pdf" order="12" />
-                <GridItem name="_t_Italy_t_" url="/regulatory/countries/Italy.pdf" order="13" />
-            </CssGridColumn>
-            <CssGridColumn>
-                <Country>
-                    <Order>14</Order>
-                    <License>
+        <div className={styles.eu_grid}>
+            <CustomLink href="/regulatory/countries/Austria.pdf" className=" inline">
+                <div className={styles.country_container}>
+                    <FlagAustriaIcon className={styles.icon} fill="#000000b8" />
```

Comment:
Can you convert this by looping in array as all the components are repeating.We can use an array something like this 
`const countries = [
  { href: "/regulatory/countries/Greece.pdf", icon: <FlagGreeceIcon className={styles.icon} iconSize="md" fill="#000000b8" />, text: "_t_Greece_t_" },
  { href: "/regulatory/countries/Hungary.pdf", icon: <FlagHungaryIcon className={styles.icon} iconSize="md" fill="#000000b8" />, text: "_t_Hungary_t_" },
  { href: "/regulatory/countries/Italy.pdf", icon: <FlagItalyIcon className={styles.icon} iconSize="md" fill="#000000b8" />, text: "_t_Italy_t_" }
];`